jobs/config.json

```diff
@@ -3514,6 +3515,7 @@
       "--extract=ci/latest",
       "--gcp-master-image=gci",
       "--gcp-node-image=gci",
+      "--gcp-node-size=g1-small",
```

Comment:
Maybe we can keep this (given that it saves us ~2x cost)?

Reply:
I don't understand. Keep what?

Reply:
Keep using g1-small nodes.

Reply:
I still don't understand - I want to use them that's why I added that.

Reply:
Oh sorry.. that's correct. I don't know what's wrong (I guess with my github UI) that I see the opposite diff.